spec.html

```diff
@@ -15898,7 +15898,7 @@ <h1>[[OwnPropertyKeys]] ( ): either a normal completion containing a List of pro
             The returned List contains no duplicate entries.
           </li>
           <li>
-            The Type of each result List element is either String or Symbol.
+            Each result List element is a property key.
```

Comment:
The status quo has "result List element", but it's an odd phrase. 
```suggestion
            Each element of the result List is a property key.
```
would be more idiomatic.